### ISSUE TITLE
🐛 [BUG] DualAxis When data for column is all nulll, column won't disppear

### ISSUE THREAD

**Issue by susiwen8** [antvis/G2Plot#2744]
### 💻 Link to minimal reproduction

https://codesandbox.io/s/cocky-wave-hor9n?file=/index.ts

### 📷 Step to reproduce

Open link you can see 
<img width="1056" alt="Screen Shot 2021-07-31 at 2 28 26 PM" src="https://user-images.githubusercontent.com/20318608/127731028-53910455-46a3-44c7-8fb9-170ae98baa96.png">
 

### 🏞 Expected result
<img width="1046" alt="Screen Shot 2021-07-31 at 2 28 43 PM" src="https://user-images.githubusercontent.com/20318608/127731036-e5d3ecfc-c031-4093-8496-29c450d4d0d0.png">

### 🚑 Any additional [like screenshots]

* **G2Plot Version**: latest
* **Platform**: MacOS Chrome

**Comment by susiwen8** [comment]
G2 don't have this problem
https://codesandbox.io/s/kind-silence-302my

**Comment by Shamann** [comment]
> G2 doesn't have this problem
> https://codesandbox.io/s/kind-silence-302my

No, G2 DOES have this problem. If you comment out axis scale config in your example (to make its config similar to the one used for G2Plot example from the issue description), you'll see there is no axis anymore, no dimention, so the y coordinate of geometries corresponding to null data points are meaningless, but geometries are still there. Please, consider this example <https://codesandbox.io/s/hungry-leftpad-oxgjl?file=/index.ts:663-715> - the same bug with columns.
I found a workaround for both G2 and G2Plot. The point is to "preserve" the hidden axis by setting scale's value to an arbitraty non-empty object.
G2: <https://codesandbox.io/s/hungry-leftpad-oxgjl?file=/index.ts:587-662> - as you can see, the data for "people" is also null, but there is no line on the chart
G2Plot: <https://codesandbox.io/s/sad-chebyshev-04p3n?file=/index.ts:420-501>
